src/jaeguk-cho/230206/호텔 대실.kt

```diff
@@ -0,0 +1,17 @@
+package `jaeguk-cho`.`호텔 대실`
+
+/**
+ * @author Jaeguk Cho
+ */
+
+class `호텔 대실` {
+    fun solution(book_time: Array<Array<String>>): Int {
+        val time = MutableList(2400) { 0 }
+        book_time.forEach { (s, e) ->
+            time[s.filter { it != ':' }.toInt()]++
+            time[e.filter { it != ':' }.toInt() + 10]--
+        }
+
+        return time.runningReduce(Int::plus).maxOf { it }
```

Comment:
```suggestion
        return MutableList(2400) { 0 }.apply {
            book_time.forEach { (s, e) ->
                time[s.filter { it != ':' }.toInt()]++
                time[e.filter { it != ':' }.toInt() + 10]--
            }
        }.runningReduce(Int::plus).maxOf { it }
```

(일차원적으로) 이렇게 줄여볼 생각을 했었는데

Reply:
이렇게 했어도 깔끔했을것 같네요 !